assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.js

```diff
@@ -55,35 +55,38 @@ function DashboardUniqueVisitorsWidget() {
 		const accountID = store.getAccountID();
 		const profileID = store.getProfileID();
 		const internalWebPropertyID = store.getInternalWebPropertyID();
-		const commonArgs = {
-			dateRange: select( CORE_USER ).getDateRange(),
-		};
 
 		const url = select( CORE_SITE ).getCurrentEntityURL();
-		if ( url ) {
-			commonArgs.url = url;
-		}
+		const { compareStartDate, compareEndDate, startDate, endDate } = select( CORE_USER ).getDateRangeDates( { offsetDays: DATE_RANGE_OFFSET, compare: true } );
+
 		const sparklineArgs = {
+			compareStartDate,
+			compareEndDate,
```

Comment:
These two shouldn't be here, the `sparklineArgs` object previously didn't have `multiDateRange: 1`, so it should only have `startDate` and `endDate` to keep that behavior. Only `args` should have the two extra properties because that had `multiDateRange: 1`.